lib/data-types.js

```diff
@@ -473,6 +546,9 @@ RANGE.prototype.key = RANGE.key = 'RANGE';
 RANGE.prototype.toSql = function() {
   return pgRangeSubtypes[this._subtype.toLowerCase()];
 };
+RANGE.prototype.validate = function(value) {
+  return true;
```

Comment:
An array with two elements

Might have an `inclusive` property which is also an array with two elements